unit_test/entities/test_ates_cluster.py

```diff
@@ -0,0 +1,79 @@
+#  Copyright (c) 2023. Deltares & TNO
```

Comment:
Tests do not run. I get an error on the import of simulator_core, so I will get back to you about that. 

The tests do not seem to be fully covering the code, but I cannot check.

Reply:
me too, seems importing jnius causing the test failed

Reply:
We need to fix that before we can commit. @samvanderzwan @MichielTukker can you help?

Reply:
Investigated what the cause is. Checked with WinDBG and it seems to be a known issue once `java.dll` is loaded by the process. It isn't an issue within jnius, Python, pytest or within your code what I can tell. Neither in the Rosim jar. Also, from what I can tell, the exception is printed but also handled properly by the underlying code (as nothing seems to crash).

The reason this is triggered only when testing is because pytest enables the Python `faulthandler` since 5.0 https://docs.python.org/3/library/faulthandler.html . While this Access exception is a known issue, all literature I can find online mentions to just disable the faulthandler to circumvent this 'seemingly-innocent' exception. Disabling the faulthandler in general seems like a bad idea (which is why Sam also asked if I could investigate). I propose the following workaround to minimally turn off the fault handler: https://github.com/Project-OMOTES/simulator-core/pull/133 .

Actually fixing this ancient error would entail investigating inside the Java dll which I believe would take longer than the benefit it would give.